centaur/src/main/resources/standardTestCases/no_output_delete.test

```diff
@@ -0,0 +1,23 @@
+# When there are no files to delete, Cromwell should successfully transition to terminal state
+
+name: no_output_delete
+testFormat: workflowsuccess
+backends: [Papiv2]
+
+files {
+  workflow: delete_intermediates/no_output_delete.wdl
+  options: delete_intermediates/delete_intermediates.options
```

Comment:
I don't see this file in this PR?

Reply:
It is already [exists](https://github.com/broadinstitute/cromwell/blob/develop/centaur/src/main/resources/standardTestCases/delete_intermediates/delete_intermediates.options) as part of previous tests.